packages/react-components/theme-designer/src/components/TokenBoxes/TokenBoxes.stories.tsx

```diff
@@ -0,0 +1,3 @@
+import { TokenBoxes } from './TokenBoxes';
+export default { component: TokenBoxes };
+export const Default = {};
```

Comment:
we CAN come back to the stories later, but note that this TokenBoxes story doesn't render without brandColors passed into it.